JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/LinkerPreserve.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Akavache.Sqlite3;
+using MvvmCross;
+
+namespace JhipsterXamarin
+{
+    /// <summary>
+    /// Linker preserve for Akavache (https://github.com/reactiveui/Akavache).
+    /// Note: This class file is *required* for iOS to work correctly.
```

Comment:
is not only for ios 
add the exact link https://github.com/reactiveui/Akavache#handling-xamarin-linker